### ISSUE TITLE
bcmonitor is telling me the insight api is returning 404

### ISSUE THREAD

**Issue by dabura667** [bitpay/bitcore-wallet-service#247]
status: 404, url: '/api/addrs/txs', error: 'Not found'

This seems to happen when I take a wallet originally registered on bitpay's server and tap the button to register on my server. Then any balance check for that wallet through my server gives me the above message.

I also tried using the GET method of the same api (Insight) with

http://<IP:PORT4INSIGHT>/api/addrs/1LbhWP7cM3ioyu6q1exUHwNbxXmpituYdm/txs?from=0&to=20

And I get a 404.

All other functions seem to be fine...

Wallet that was created on my server seems fine, but then all of a sudden iPhone starts giving me errors saying I can't check my balance when I try to tap and refresh. But when I close the app and reopen it, it shows the balance.

Meanwhile, Android is fine. tapping the balance updates it fine, no need to retsart the app.

Versions:
bws  "version": "0.0.33",
insight   "version": "0.2.5",
insight-bitcore-api    "version": "0.2.11",

To summarize:
1. When registering bitpay server wallet to my server, both Android and iPhone have trouble grabbing history. nohup.out shows POST 404 errors on /api/addrs/txs
2. When just checking balance for wallet made on my server with iPhone (2 of 2 with Android) it only shows balance on first load. (this is updated each time I restart the app) But periodic automated updates and manual updates (tap balance) will load forever and throw an error, leaving nohup.out on my server to say POST response 404 on /api/addrs/txs again.

**Comment by dabura667** [comment]
I updated to insight api 0.2.13 and now I have api/addrs/txs and it works fine, no more 404 errors.

However, my iPhone still has a hard time looking up balances for wallets that were'nt originally made on my server.

Android is perfectly fine.

iPhone:
1. Tap back and forth between wallets originally created on my server. Wallet balance update is quick and smooth.
2. Tap on any wallet originally made on bitpay servers which I registered to my server.
3. iPhone no longer is able to check the balance for any wallet and it is showing errors on balance screen.
4. Select one of the my-server wallets, then kill the app on iPhone.
5. Restart the app.
6. Start over from 1

This might be remedied by upgrading iOS, as iirc iOS is a lower version than Android.

Funny that the problem persists on iPhone, no error log in nohups for locker messagebroker bcmonitor emailservice bws insight all show no errors, and only show the successful GETs that are 200 response.

**Comment by dabura667** [comment]
The opposite on iPhone is fine:

If I register one of my wallets with your server, then the scan is very quick and the balance loads quick and smoothly after the scan is done.

Maybe the registration process failed and I have corrupted data on my db?...

I have to go to bed. Until I can fix this I will just stick to Android for now.

**Comment by dabura667** [comment]
Now my iPhone is perfectly fine...

I can switch back and forth between wallets rapidly just like Android could last night.

Android still works fine.

I will leave this open because someone should add a console warning in BWS that verifies the version number of the insight-api being queried and warns if the version is too old.

**Comment by dabura667** [comment]
Now my iPhone is same as last night.

I wonder why it's only my iPhone...
